### PR TITLE
fix problems with multi converter

### PR DESCRIPTION
**Pull request by littlerobinson**
Fix some problems with multi concerter:

- user can choose the youtube quality before the loop on the list of youtube content
- the blanks lines in the links file are ignored
- write the text file with the title of youtube content

### CODE REVIEW

**Comment by littlerobinson** [comment]
Hello I put some change to improve multi coneverter, thx

File: multi_conv/multi_converter.py [hunk 1]
```diff
@@ -73,22 +72,30 @@ def main():
                 print(f"Directory '{directory}' created successfully.")
 
     # Read links from the link_file
-    with open(link_file, 'r') as link_file:
-        links = [line.strip() for line in link_file]
+    with open(link_file, 'r') as file:
+        links = [line.strip() for line in file if line not in ['\n', '\r\n']]
```
Comment (daniau23) [inline multi_conv/multi_converter.py:76]: checking for empty lines in the text file?

File: multi_conv/multi_converter.py [hunk 2]
```diff
@@ -73,22 +72,30 @@ def main():
...
 
     # Get the list of downloaded MP4 files
-    mp4_files = [f for f in os.listdir(mp4_dir) if f.endswith('.mp4')]
+    mp4_names = [f for f in os.listdir(mp4_dir) if f.endswith('.mp4')]
+
+    # Write the mp3 titles in the name_file
+    with open(name_file, "w") as file:
+        for line in mp4_names:
+            print("line", line)
+            file.write(line+"\n")
```
Comment (daniau23) [inline multi_conv/multi_converter.py]: There should be a way to remove ".mp4"  to avoid saving  it as file_name.mp4.mp3. Kindly address for final merge.

**Review by daniau23 — CHANGES_REQUESTED** [review]
Lines 84 - 90
There should be a way to remove ".mp4" to avoid saving it as file_name.mp4.mp3. Kindly address for final merge.

File: multi_conv/multi_converter.py (hunk 1 above)
Comment (littlerobinson) [reply multi_conv/multi_converter.py:76]: yes

**Review by littlerobinson — COMMENTED** [review]
Ok i make a new change to delete mp4 suffix

**Review by daniau23 — APPROVED** [review]
@littlerobinson Thank you for your contributions to this project.

**Pull request merged**